packages/header/src/sass/_module.scss

```diff
@@ -0,0 +1,56 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// header module
+// Headers provide a consistent start to pages.
+//
+// Content:
+// - Header styles
+//   - logo style
+//     - site title style
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Header styles
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Basic header block styling.
+ */
+.uikit-header {
+	padding: uikit-space( 0.5 ) uikit-space( 1 ) uikit-space( 1 );
+	background-image: linear-gradient(to top, #f0f3f5, #fff);
+	border-bottom: 6px $uikit-colour-Border-40 solid;
+	margin-bottom: uikit-space( 1 );
+
+	/**
+	 * Image logo styling -- indented to be an anchor back to the homepage.
+	 */
+	.uikit-header__logo {
+		border-bottom: none;
+
+		&:hover,
+		&:focus {
+			background-color: transparent;
+		}
+
+		/**
+		 * Site title styling
+		 */
+		.uikit-header__logo__site-title {
```

Comment:
You don't need to nest at all here. Let's make these classes flat so specificity stays easy ;)